app/services/transaction_service/transaction.rb

```diff
@@ -48,22 +48,14 @@ def can_start_transaction_braintree(community_id:, author_id:)
   end
 
   def can_start_transaction_paypal(community_id:, author_id:)
-    person_account_response = paypal_account_api.get(
-      community_id: community_id,
-      person_id: author_id
-    )
-    personal_account_active = Maybe(person_account_response)[:data][:active]
-
-    admin_account_response = paypal_account_api.get(
-      community_id: community_id
-    )
-    admin_account_active = Maybe(admin_account_response)[:data][:active]
+    personal_account_prepared = PaypalHelper.account_prepared_for_user?(author_id, community_id)
```

Comment:
Err... I think this is not right. Transaction service doesn't know about PaypalHelper, which is a module in the "main" web service.